docs/src/modules/constants.js

```diff
@@ -4,10 +4,10 @@ const CODE_VARIANTS = {
 };
 
 // Valid languages to server-side render in production
-const LANGUAGES = ['en', 'zh', 'pt'];
+const LANGUAGES = ['en'];
```

Comment:
Do we need these constants? They're used in `next.config.js`, but maybe we should remove translation logic there as well?

Reply:
We discussed that we want to keep the infrastructure around the translations so that we can easily add it back if we decide in the future, this is why I decided to left these constants and not do any changes in the `next.config.js`, but I don't mind removing the logic there as well. cc @siriwatknp what do you think?